packages/core/src/components/Dialog/Dialog.tsx

```diff
@@ -171,7 +171,7 @@ export default class Dialog extends PureComponent<DialogProps, DialogState> {
   static positions = DialogPosition;
   static animationTypes = AnimationType;
   static defaultProps = {
-    position: "top",
+    // position: "top",
```

Comment:
is it on purpose? what's the reason if yes?

Reply:
mistake, fixed